scripts/unlink.sh

```
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+# Remove the symlinks that link.sh created
+# Only links that still point into this repo are touched
+
+source "$DOTFILES_DIR/scripts/vars.sh"
```

Shellcheck (0.11.0)

[info] Not following: ./scripts/vars.sh was not specified as input (see shellcheck -x).

(SC1091)